ui/src/app/customizations/tech-extensions/tech-extensions-list.component.ts

```diff
@@ -10,19 +11,22 @@ import {
   ListEvent,
   EmptyStateConfig,
 } from 'patternfly-ng';
-import { Extensions } from '../../model';
+import { Extensions, Extension } from '../../model';
 
 @Component({
   selector: 'syndesis-tech-extensions-list',
-  templateUrl: 'tech-extensions-list.component.html'
+  templateUrl: 'tech-extensions-list.component.html',
+  styleUrls: ['./tech-extensions-list.component.scss']
 })
 export class TechExtensionsListComponent implements OnInit {
   extensions: Observable<Extensions>;
```

Comment:
The convention for observable fields and variables is to name them with a trailing `$` (`extensions$` in this case), so one can recognize at a glance that this variable or that class property is observable and hence requires to be handled with special care.